File: VB/Controllers/HomeController.vb
Imports System.IO
Imports System.Web
Imports System.Web.Mvc
Imports DevExpress.XtraPrinting
Imports DevExpress.XtraPrintingLinks
Imports DevExpress.Web.Mvc
Imports DevExpress.Web.ASPxGridView
Imports System.Data
Imports DevExpress.XtraReports.UI
Imports System.Collections.Generic
Imports DevExpress.Data.Filtering


Namespace CS.Controllers
    Public Class HomeController
        Inherits Controller
        Public Function Index() As ActionResult
            ViewData("CmbValue") = "Blue"
            Return View(LargeDatabaseDataProvider.GetColors())
        End Function

        Public Function DataBindingToLargeDatabasePartial() As ActionResult
            ViewData("CmbValue") = Request.Params("cmbValue")
            Return PartialView("GridViewPartial")
        End Function
    End Class

End Namespace

File: CS/Controllers/HomeController.cs
using System;
using System.Web.Mvc;
using System.Collections.Generic;
using DevExpress.Web.Mvc;


namespace CS.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            ViewData["CmbValue"] = "Blue";
             return View(LargeDatabaseDataProvider.GetColors());
        }

        public ActionResult DataBindingToLargeDatabasePartial() {
            ViewData["CmbValue"] = Request.Params["cmbValue"];
            return PartialView("GridViewPartial");
        }
    }
}
